contracts/Vault1.sol

```diff
@@ -1,6 +1,25 @@
-//SPDX-License-Identifier: Unlicense
-pragma solidity ^0.8.4;
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
```

Comment:
You can use the ERC20 interface rather than using the complete ERC20 implementation.